pkg/noderesourcetopology/filter.go

```diff
@@ -70,6 +70,12 @@ func resourcesAvailableInAnyNUMANodes(logKey string, numaNodes NUMANodeList, res
 	nodeName := nodeInfo.Node().Name
 
 	for resource, quantity := range resources {
+		if quantity.IsZero() {
```

Comment:
I still cannot figure out how a user can ask for zero resources, is that even possible when you're filling in the pod spec?

Reply:
not for cpu and memory, but kube 1.24.0 accepts pod requesting 0 devices.